UI/src/app/pages/mapping/components/dialog/dialog.component.ts

```diff
@@ -0,0 +1,115 @@
+import { Component, OnInit, ViewChild, Renderer2, AfterViewInit, OnDestroy, ViewEncapsulation, TemplateRef } from '@angular/core';
+import { OverlayRef } from '@angular/cdk/overlay';
+import { CommentsService } from 'src/app/pages/mapping/services/comments.service';
+import { CommonService } from 'src/app/pages/mapping/services/common.service';
+import { Store, select } from '@ngrx/store';
+
+export interface IComment {
+  id: number;
+  text: string;
+  date: Date;
+  hasBeenEdited: boolean;
+}
+
+export class Comment {
+  id: number;
+  text: string;
+  date: Date;
+  hasBeenEdited: false;
+
+  constructor(id: number, text: string) {
+    this.id = id;
```

Comment:
no need in ```this.id = id; this.text = text;``` inside of the constructor.
You can read about it [there](https://kendaleiv.com/typescript-constructor-assignment-public-and-private-keywords/)

Reply:
i thought explicit version would be more readable)